### ISSUE TITLE
Charges - Workflow

### ISSUE THREAD

**Issue by arakla** [sc0v/binder-app#201]
After approving a charge from an org charge page, binder loads the charges index as opposed to the org charge page you were just on

**Comment by blinblinblin** [comment]
@arakla @pkoenig10 Is it okay that we redirect to the organization page as opposed to the organization's charges tab? The latter increases the complexity of the issue.

**Comment by arakla** [comment]
In my opinion, that would be ok since leaving the org page is a bigger hassle. However, in terms of workflow, it is still a pain if you are trying to review many charges at once.

Side tangent for the workflow:
Ideally, you could hover over a charge to view details or go to the show page. And from either an org page or charges index page, you could approve a charge (which should already be doable from the charges index).

The part that would make all of binder better: Be able to do things like approve charges, without the whole page refreshing...AJAX maybe?

**Comment by arakla** [comment]
Just realized that for my Ajax comment, #213 covers it

**Comment by pkoenig10** [comment]
Agreed that AJAX is the right way to handle this.  Updating the approval workflow to use AJAX shouldn't be too much lift.

And ideally, in any workflow, we'd like to take the user back to the previous page, wherever that was.  That way we don't care what pages link where (because that will certainly change in the future).  Rails has the ability to go back to the previous page, although it might take some tweaking to get the behavior correct for multi-page workflow.  But then I would encourage you to replace these multipage workflow with AJAX.  I think in general, multipage workflows are what slow down alot of Binder use.

**Comment by ChaseBro** [comment]
Also we really need to just start actually implementing modals, raw rails makes that kind of hard, but we should figure it out regardless
